src/deployments/procedures/roles/AaveV4SpokeRolesProcedure.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: UNLICENSED
+// Copyright (c) 2025 Aave Labs
+pragma solidity ^0.8.0;
+
+import {Roles} from 'src/libraries/types/Roles.sol';
+import {IAccessManager} from 'src/dependencies/openzeppelin/IAccessManager.sol';
+import {ISpoke} from 'src/spoke/interfaces/ISpoke.sol';
+
+library AaveV4SpokeRolesProcedure {
+  function grantSpokeConfiguratorRole(
+    address accessManagerAddress,
+    address spokeConfiguratorAddress
+  ) internal {
+    IAccessManager(accessManagerAddress).grantRole({
+      roleId: Roles.SPOKE_CONFIGURATOR_ROLE,
+      account: spokeConfiguratorAddress,
+      executionDelay: 0
+    });
+  }
+
+  function grantSpokeAdminRole(address accessManagerAddress, address spokeAdminAddress) internal {
```

Comment:
these fn is before grant configurator role for hub. let's try to use the same ordering